server/importers/MultiImporter.ts

```diff
@@ -0,0 +1,260 @@
+import { existsSync, createWriteStream, mkdirSync, writeFileSync } from 'fs'
+import { resolve as _resolve, join } from 'path'
+import * as rimraf from 'rimraf'
+import axios from 'axios'
+import * as extract from 'extract-zip'
+import { promisify } from 'util'
+import { red } from 'colors'
+import log from '../logger.js'
+import GtfsImport from '../db/gtfs-import.js'
+import CreateShapes from '../db/create-shapes.js'
+import connection from '../db/connection.js'
+import Storage from '../db/storage.js'
+import KeyvalueDynamo from '../db/keyvalue-dynamo.js'
+import config from '../config'
+
+const shapeFile = 'shapes.txt'
+
+interface IMultiImporterProps {
+  keyvalue?: string
+  keyvalueVersionTable?: string
+  keyvalueRegion?: string
+  locations: { endpoint: string; type: string; name: string }[]
+  downloadInterval?: number
+  batchSize?: number
+  authorization?: string
+}
+
+abstract class MultiImporter {
```

Comment:
Quite a lot of the multi-importer is the same as the baseimporter. Maybe have one base class, and then extend it with the BaseImporter and MultiImporter? Up to you.